README.md

```diff
@@ -183,3 +187,4 @@ Tip: If your build hangs, try to increase the memory available to Docker.
 [CaptureUndistortRGB-url]: source/Applications/Advanced/CaptureUndistortRGB/CaptureUndistortRGB.cpp
 [CreateDepthMap-url]: source/Applications/Advanced/CreateDepthMap/CreateDepthMap.cpp
 [MaskPointCloud-url]: source/Applications/Advanced/MaskPointCloud/MaskPointCloud.cpp
+[TransformViaArucoMarker-url]: source/Applications/Advanced/TransformViaArucoMarker/TransformViaArucoMarker.cpp
```

Comment:
TransformViaArucoMarker - I am wondering if we should rename the sample to TransformPointCloud or TransformPointCloudViaArucoMarker.

The reason is that 'transform' on its own can mean anything. The description of the sample is great and clear, but can we do something to improve the sample name as well? We started with 'transform' because that is how our feature is called. Shold we consider 'change of origin' or 'point cloud origin change' or something like that instead? Let's brainstorm a bit during the review.

Whatever you decide don't implement before the review is finished.

Reply:
I agree that 'Transform' can mean anything, but with the rest of the name (pointCloudUsingArucoMarker), i think it puts it in context. 

If we are going to change it then I like 'TransformPointCloudViaArucoMarker' or 'TransformPointCloudUsingArucoMarker'.

Reply:
I agree we should mention pointcloud and aruco in the name. I like both suggestions from christians last comment.

Reply:
I think that is sufficient enough. Whatever you decide don't implement before the review is finished.

Reply:
OK, i wont do change it as of now. I created a task for remaining me in the PR message on the top of the page.

Reply:
I am implementing this in the other PR: https://github.com/zivid/zivid-cpp-samples/pull/165